OpenRA.Game/Network/Connection.cs

```diff
@@ -66,13 +66,15 @@ public virtual void Send(int frame, List<byte[]> orders)
 			Send(ms.ToArray());
 		}
 
-		public virtual void SendImmediate(List<byte[]> orders)
+		public virtual void SendImmediate(IEnumerable<byte[]> orders)
 		{
-			var ms = new MemoryStream();
-			ms.WriteArray(BitConverter.GetBytes(0));
 			foreach (var o in orders)
+			{
+				var ms = new MemoryStream();
```

Comment:
I just wondered why we're not disposing this. I suppose the GC will collect that sooner or later, but since we are creating multiple streams now, it might make sense to warp this in a `using` statement. (Or doesn't it?)

Reply:
MemoryStream's dont use any unmanaged resources, so in practice this shouldn't matter either way.

This is just moving existing code inside a block, so IMO shouldn't be a blocker.

Reply:
I don't mind doing the change, but it should be consistent across the whole codebase. Either all 63 usages of MemoryStream should be wrapped in `using` or none in my opinion.

Reply:
I agree, and that's a bit out of scope for this PR and could be done in a follow-up, if we decide to go for it.